extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ParquetFileWriter.java

```diff
@@ -78,22 +77,21 @@ public void close() throws IOException {
         serializeOffsetIndexes();
         final ParquetMetadata footer =
                 new ParquetMetadata(new FileMetaData(type, extraMetaData, Version.FULL_VERSION), blocks);
-        serializeFooter(footer, bufferedOutput);
-        metadataFileWriter.addParquetFileMetadata(destFilePathForMetadata, footer);
-        // Flush any buffered data and close the channel
-        bufferedOutput.close();
+        serializeFooter(footer, countingOutput);
+        metadataFileWriter.addParquetFileMetadata(destForMetadata, footer);
+        // Flush any buffered data, do not close the stream since it is managed by the calling code
+        countingOutput.flush();
```

Comment:
It's not clear to me that we need / want to flush here if it's managed by the caller.